src/client.rs

```diff
@@ -4,7 +4,7 @@ use serde_json::{json, Value};
 use std::collections::HashMap;
 
 /// The top-level struct of the SDK, representing a client containing [indexes](../indexes/struct.Index.html).
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct Client {
     pub(crate) host: Rc<String>,
     pub(crate) api_key: Rc<String>,
```

Comment:
Since this `Rc` doesn't refer to `std`'s `Rc` (see [here](https://github.com/meilisearch/meilisearch-rust/blob/70a0d4014cfc2c098ff0124b6cee32c4051cd4c5/src/lib.rs#L238-L241)), to me it seems that adding an `Arc`/`Rc` is unnecessary; and the user can always just add them still if they find they need them.

Reply:
It is indeed a custom type defined in lib.rs. Depending on a feature, it can either resolve to Rc or Arc. In the past, we only used std Rcs. Then I added this to allow the user to make our Rcs atomic. It was Rc by default in the past but has been changed to Arc since. Maybe we could rename it